ignite/distributed/comp_models/base.py

```diff
@@ -275,8 +277,24 @@ def _do_all_gather(self, tensor: torch.Tensor) -> torch.Tensor:
     def _do_broadcast(self, tensor: torch.Tensor, src: int) -> torch.Tensor:
         pass
 
+    def _check_barrier_fn_kwargs(self, barrier_fn: Callable, kwargs_dict: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
This function is more general than its name suggests. You did a signature checker. It could be called `check_method_args` and note that type could be also checked.